android/src/main/java/com/robinpowered/react/vitals/RNVitalsModule.java

```diff
@@ -70,7 +70,9 @@ private WritableMap getMemoryInfo() {
   public void onTrimMemory(int level) {
     ReactApplicationContext context = getReactApplicationContext();
     if (context.hasActiveCatalystInstance()) {
-      context.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class).emit(LOW_MEMORY, getMemoryInfo());
+      WritableMap memoryInfo = getMemoryInfo();
+      memoryInfo.putInt("memoryLevel", level);
```

Comment:
What's the memory level mean here?

Is it the amount of memory used by the app at the time of the warning? Is it a level of urgency? etc